src/webextension/background/datastore.js

```diff
@@ -4,6 +4,11 @@
 
 import * as DataStore from "lockbox-datastore";
 
-const datastore = DataStore.create({});
+let datastore;
 
-export default datastore;
+export default async function openDataStore() {
+  if (!datastore) {
+    datastore = (typeof DataStore.open === "function") ? await DataStore.open() : DataStore.create();
```

Comment:
We could probably wait to change this until we update our dependency on lockbox-datastore. We'll want to change this line anyway once we update, since we don't care about backwards-compatibility in this way. It doesn't matter too much for now though...

Reply:
🐔 <==> 🥚